packages/webapp/src/components/Task/TaskReadOnly/index.js

```diff
@@ -206,12 +187,8 @@ export default function PureTaskReadOnly({
               />
             </div>
           )}
-          {(!task.happiness) && (
-            <Checkbox
-              label={t('TASK.PREFER_NOT_TO_SAY')}
-              disabled={true}
-              defaultChecked={true}
-            />
+          {!task.happiness && (
+            <Checkbox label={t('TASK.PREFER_NOT_TO_SAY')} disabled={true} defaultChecked={true} />
```

Comment:
for boolean props, we can just put the prop name for true.
e.g.: `disabled={true}` -> `disabled`